exercises/retree/canonical-data.json

```diff
@@ -1,66 +1,66 @@
 {
-    "exercise": "retree",
-    "version": "1.0.0",
-    "cases": [
-      {
-        "description": "Empty tree",
-        "property": "treeFromTraversals",
-        "input": {
-          "preorder": [],
-          "inorder": []
-        },
-        "expected": {}
+  "exercise": "retree",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Empty tree",
+      "property": "treeFromTraversals",
+      "input": {
+        "preorder": [],
+        "inorder": []
       },
-      {
-        "description": "Tree with one item",
-        "property": "treeFromTraversals",
-        "input": {
-            "preorder": ["a"],
-            "inorder": ["a"]
-        },
-        "expected": {"v":"a", "l": {}, "r": {}}
+      "expected": {}
+    },
+    {
+      "description": "Tree with one item",
+      "property": "treeFromTraversals",
+      "input": {
+          "preorder": ["a"],
+          "inorder": ["a"]
       },
-      {
-        "description": "Tree with many items",
-        "property": "treeFromTraversals",
-        "input": {
-            "preorder": ["a", "i", "x", "f", "r"],
-            "inorder": ["i", "a", "f", "x", "r"]
-        },
-        "expected": {"v": "a",
-            "l": {"v": "i", "l": {}, "r": {}},
-            "r": {"v": "x",
-                "l": {"v": "f", "l": {}, "r": {}},
-                "r": {"v": "r", "l": {}, "r": {}}
-            }
-        }
+      "expected": {"v":"a", "l": {}, "r": {}}
```

Comment:
don't forget this one, where there is missing a space after the colon in `"v":"a"`

(manually found, so I may have missed others)

Reply:
Ah yes, good catch. @fidelcoria could you perhaps also fix this?

Reply:
Not a problem. there were no other similar occurrences (checked with regex ":[^\s]")